knowledge/example_test.go

```diff
@@ -56,15 +57,20 @@ func ExampleClient_Read() {
 		_ = client.Close()
 	}()
 
-	path := "(n:Person)-[:HAS]->(s:Store)"
-	conditions := "WHERE n.external_id = $external_id"
+	query := "(n:Person)-[:HAS]->(s:Store) WHERE n.external_id = $external_id"
 	params := map[string]*knowledgepb.InputParam{
 		"external_id": {
 			Value: &knowledgepb.InputParam_StringValue{StringValue: "1234"},
 		},
 	}
+	returns := []*knowledgepb.Return{
+		{
+			Variable:   "n",
+			Properties: []string{},
```

Comment:
this can just be unspecified/nil if you don't want to provide properties

Reply:
updated